app/components/questions/BelongingRates.js

```diff
@@ -57,6 +60,48 @@ function transformData(response, error, selectedArea) {
 
 function ExamplePieChart(props) {
   const { data } = props
+  
+  
+  const tooltipStyle = {
+    backgroundColor: "white", 
+    borderColor: "LightGrey", 
+    borderStyle: "solid", 
+    borderWidth: "thin"
+  };
+  
+  
+  const CustomTooltip = ({ active, payload, label }) => {
+  if (active && payload && payload.length) {
+    // Changes the tooltip to display the formatted percentage rather than a decimal
+    return (
+      <div className="custom-tooltip" style={tooltipStyle}>
+        <p className="label">{`${payload[0].name} : ${(payload[0].value * 100).toFixed(1)}%`}</p>
+      </div>
+    );
+  }
+  return null;
+  };
+  
+  
+  const RADIAN = Math.PI / 180;
+  const renderCustomizedLabel = ({ cx, cy, midAngle, innerRadius, outerRadius, color, valueFormatted, index }) => {
```

Comment:
Believe it or not, as far as I know, this is the intended way in Recharts to customize labels and tooltips. Great job researching and learning it!

Each page can have its own custom labels and custom field components. If we have resuable custom components, we can move them into common files so others can import them.